src/main/groovy/com/ullink/gradle/nunit/NUnit.groovy

```diff
@@ -215,6 +228,7 @@ class NUnit extends ConventionTask {
         prepareExecute()
 
         def mbr = project.exec {
+            environment env
```

Comment:
any idea (I didn't check) if when env is empty this will change the behavior from the previous plugin version (still inherits process env variables right ?)